metricbeat/module/prometheus/collector/collector.go

```diff
@@ -81,6 +91,16 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	}
 
 	for _, family := range families {
+		if m.includeMetrics != nil {
+			if !matchMetricFamily(*family.Name, m.includeMetrics) {
```

Comment:
Not sure if this can happen, but should we check if `familiy` is nil before dereferencing it?

Reply:
I'm not sure either, but it would be safer to check it yeah!